Doc/library/pathlib.rst

```diff
@@ -540,8 +540,10 @@ Pure paths provide the following methods and properties:
 
 .. method:: PurePath.relative_to(*other)
 
-   Compute a version of this path relative to the path represented by
-   *other*.  If it's impossible, ValueError is raised::
+   Return the relative path to another path identified by *other*, each
+   element of *other* can be either a string representing a path segment,
+   a :class:`PurePath` or a subclass of :class:`PurePath`. If the operation
```

Comment:
Are you sure the path segment is `PurePath`?

Reply:
The description here is inaccurate，I will revise it as soon